cmd/covd/key.go

```diff
@@ -66,14 +66,31 @@ func createKey(ctx *cli.Context) error {
 	backend := ctx.String(keyringBackendFlag)
 	passphrase := ctx.String(passphraseFlag)
 	hdPath := ctx.String(hdPathFlag)
-	keyBackend := ctx.String(keyringBackendFlag)
+
+	if backend != "test" {
+		return fmt.Errorf("the command-line keyring backend should be 'test'")
+	}
```

Comment:
let's not enforce `test` in creating keys, but enforce it in `start`. This would allow the user to create a key in other keyring for better security